src/category_theory/comma.lean

```diff
@@ -4,6 +4,7 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Scott Morrison, Johan Commelin
```

Comment:
Add yourself to the author line in files where you do anything non-trivial! It's credit, but also helpful if someone later wants to get help understanding something.

Reply:
Done, thanks!